bucket_manifest/object_metadata_job.py

```diff
@@ -0,0 +1,118 @@
+"""
+Module for computing s3 object metadata.
+"""
+
+import sys
+import os
+import time
+import boto3
+import logging
+import hashlib
+import json
+from urllib.parse import unquote_plus
+from botocore.exceptions import ClientError
+
+
+logging.basicConfig(level=logging.INFO)
+logging.getLogger().addHandler(logging.StreamHandler(sys.stdout))
+
+CHUNK_SIZE = 1024 * 1024 * 10
```

Comment:
how did you decide on these numbers? maybe a quick comment about the limitations around this would be useful

Reply:
Yeah, it is not easy to get perfect number since it depends on network bandwidth, cpu type ..etc. I have just made it configurable with default value is `1024 * 1024 * 10` which is my own experience